skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -100,6 +213,13 @@ void MpActor::OnEquip(uint32_t baseId)
     VarValue args[] = { VarValue(std::make_shared<EspmGameObject>(lookupRes)),
                         VarValue::None() };
     SendPapyrusEvent("OnObjectEquipped", args, std::size(args));
+
+    if (baseId == 0x00064B43) {
```

Comment:
please check if `SweetPie.esp` is loaded (present in espmFiles)

Reply:
Hmm, making the logic based on esp names looks strange... maybe a separate switch? Or it would be an overkill?

Reply:
@nic11 don't understand